### ISSUE TITLE
Very difficult to find clothes in the handbook

### ISSUE THREAD

**Issue by MonsterHorns** [anegostudios/VintageStory-Issues#8863]
### Game Version

v.1.22.0-rc.6

### Platform

None

### Modded

Vanilla

### SP/MP

None

### Description

Clothes don't have their slot stated in the handbook, and their names are not uniform, so you can't for example just search for "shirt" and get all the shirts in the game, because some of them are called "blouse" or something else, so you need to know the item's names beforehand. You also can't easily see what you can craft from your materials, because the clothing items cycle in the handbook: to see all clothes you can craft from a small pelt for example, you have to wait for it to cycle to a new item each time you return to the pelt page. The cycling helps when the items are identical apart from the material, but is very unhelpful for clothes. There should be a way to disable cycling, if there's no universal solution for all cases.
I also think there should be a way to filter the craftable and uncraftable clothes and items in general, as with each new batch of items added it becomes harder to know what you can craft and what you can only find/buy.

### How to reproduce

1. Open a world in survival mode
2. Open the handbook
3. Try to find all items that can be equipped in the shoulder slot
4. Try to see which items you can craft with a sewing kit
5. Open a page for grey leather and try to find an ornate shield of specific color and metal

### Logs (Required)

Log files are now required in all cases (unless it's a website issue)
You can drag and drop log files here, you can also cut and paste from any crash report here: ```Crash```

**Comment by tyronx** [comment]
>  so you can't for example just search for "shirt" and get all the shirts in the gam

For that you can search for "upper body" - the items clothing category.

Also this is not a bug report, but a feature request.
